client/src/routes/user/ListPage.tsx

```diff
@@ -221,6 +267,14 @@ function isMotstandenMail(email: string): boolean {
     return email.trim().toLowerCase().endsWith("@motstanden.no")
 }
 
+function isActive(status: string): boolean {
+    return status == UserStatus.Active
+}
+
+function isRetired(status: string): boolean {
+    return status == UserStatus.Retired
+}
+
```

Comment:
Føler ikke helt at disse to funksjonene er nødvendig. Her er logikken så enkel at det burde gjøres inline